#==================================================================================================
# Patch to add option 'lzo2', to allow explicitly disabling feature. Also prevents opportunistic
# use.
#
# Author: Christopher Nielsen (GitHub: @mascguy)
#==================================================================================================
--- meson.build.orig
+++ meson.build
@@ -204,7 +204,7 @@
   endif
 endif
 
-lzo_dep = dependency('lzo2', required: false)
+lzo_dep = dependency('lzo2', required: get_option('lzo2'))
 if lzo_dep.found()
   conf.set('HAVE_LZO', 1)
 endif
--- meson_options.txt.orig
+++ meson_options.txt
@@ -23,6 +23,7 @@
 option('spectre', type : 'feature', value : 'auto')
 option('symbol-lookup', type: 'feature', value : 'auto',
        description: 'Symbol lookup in debug utils via binutils/bfd')
+option('lzo2', type : 'feature', value : 'auto')
 
 # Documentation
 option('gtk_doc', type : 'boolean', value : false,
